src/service.js

```diff
@@ -1,4 +1,15 @@
-const fetchEnterpriseCustomers = (apiClient, lmsBaseUrl = process.env.LMS_BASE_URL) => apiClient.get(`${lmsBaseUrl}/enterprise/api/v1/enterprise-customer/`);
+import qs from 'querystring';
+
+const fetchEnterpriseCustomers = (
```

Comment:
Nit: `fetchPreferredEnterpriseCustomer`?  Also, since this will only ever return 0 or 1 things, should we have this do the `.pop()`?